docs/cep/tutorials/executing-scripts.md

```diff
@@ -62,15 +62,15 @@ The following parameters are used to configure the function definition:
 
 To write custom function calls, follow the procedure below:
 
-1. Open the GUI. Click on `Stream Apps` tab.
+1. Open the GUI. Click on `Stream Workers` tab.
```

Comment:
```suggestion
1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
1. Click the **Stream Workers** tab.
```

Reply:
Refer to the Google Developers Documentation Style Guide for information on formatting.

Generally, when you click something, it should be bold, not code format.